src/neuroconv/nwbconverter.py

```diff
@@ -166,13 +166,34 @@ def create_nwbfile(self, metadata: Optional[dict] = None, conversion_options: Op
         self.add_to_nwbfile(nwbfile=nwbfile, metadata=metadata, conversion_options=conversion_options)
         return nwbfile
 
-    def add_to_nwbfile(self, nwbfile: NWBFile, metadata, conversion_options: Optional[dict] = None) -> None:
+    def add_to_nwbfile(self, nwbfile: NWBFile, metadata, conversion_options: Optional[dict] = None) -> NWBFile:
+        """
+        Add data from the instantiated data interfaces to the given NWBFile.
+
+        Parameters
+        ----------
+        nwbfile : NWBFile
+            The NWB file object to which the data from the data interfaces will be added.
+        metadata : dict
+            The metadata dictionary that contains information used to describe the data.
+        conversion_options : dict, optional
+            A dictionary containing conversion options for each interface, where non-default behavior is requested.
+            Each key corresponds to a data interface name, and the values are dictionaries with options for that interface.
+            By default, None.
+
+        Returns
+        -------
+        nwbfile : NWBFile
+            The NWB file object with the data from the data interfaces added to it.
+        """
         conversion_options = conversion_options or dict()
         for interface_name, data_interface in self.data_interface_objects.items():
             data_interface.add_to_nwbfile(
                 nwbfile=nwbfile, metadata=metadata, **conversion_options.get(interface_name, dict())
             )
 
+        return nwbfile
```

Comment:
I'm not sure why you added a return value to add_to_nwbfile (it may need one, idk), but it definitely should not be in this PR.

Reply:
Agree.

Reply:
Done